docs/running/cloud/amazon.rst

```diff
@@ -191,8 +204,11 @@ Preemptability
 ^^^^^^^^^^^^^^
 
 Toil can run on a heterogeneous cluster of both preemptable and non-preemptable nodes.
```

Comment:
We should really explain what preemptible nodes are here.  Something simple about them being discounted-cost nodes and the bidding system.  Also something warning about nodes being shut down if the spot instance pricing ends.

Reply:
Added a better description.